drv/stm32h7-spi-server/src/main.rs

```diff
@@ -528,7 +528,7 @@ fn activate_mux_option(
             pins.pin_mask,
             gpio_api::Mode::Alternate,
             gpio_api::OutputType::PushPull,
-            gpio_api::Speed::High,
+            gpio_api::Speed::Low,
```

Comment:
Hm. This really ought to be configurable, but, this is more likely to work than what we had before as long as people stay below about 10MHz.